cmd/kpod/push.go

```diff
@@ -103,7 +103,7 @@ func pushCmd(c *cli.Context) error {
 		return err
 	}
 
-	options := libkpodimage.CopyOptions{
+	options := images.CopyOptions{
```

Comment:
Using the wrong CopyOptions here - you copied it into libpod, so use the one from there, not libpod/images

Reply:
I am planning on fixing this in a different pull request, I just wanted to get everything to compile as I move stuff around.